scalyr_agent/builtin_monitors/tests/kubernetes_monitor_test.py

```diff
@@ -299,10 +301,20 @@ def wait_until_request_finished(self, pod_namespace, pod_name):
             finally:
                 self.__lock.release()
 
+        def add_pod_to_cache( self, pod_namespace, pod_name ):
```

Comment:
Recommend self-documenting rename of this function to capture semantics signifying we are simulating someone else populating the cache for us instead of the cachewarmer thread itself doing so (the idea being to speed up readability). 

Maybe something like `simulate_another_thread_adding_pod_to_cache` or something like that.

Alternatively just add docstring explaining the purpose of this function to support the test case.

Reply:
Renamed it to 'simulate_add_pod_to_cache' and added a docstring also.